elastic/kibana
# [Fleet] fix for kuery error on Agent list UI

## Summary

Fix https://github.com/elastic/kibana/issues/160493

Fixed logic that queried managed agents to exclude them from bulk actions. It was not working properly if there are no managed agent policies.

To verify:
- Enroll 10 agents with horde
- Limit rows per page to 5
- Select an agent policy that has more then 5 agents
- Select everything on all pages
- Check Network tab in Chrome Dev Tools that there is no error anymore in /agents API call

<img width="2086" alt="image" src="https://github.com/elastic/kibana/assets/90178898/d8292a54-d443-4c0d-9111-f37975529561">



### Checklist
